portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.time.format.DateTimeFormatter;  
+import java.time.LocalDateTime;
+import java.time.format.FormatStyle;
+
+// Class representing a comment
+public class Comment {
+    private String name;
+    private String comment;
+    private String date;
+
+    public Comment(){
+        name="";
+        comment="";
+        date="";
+    }
+    public Comment(String n, String c){
+        name = n;
+        comment = c;
+        date = getCurrentDate();
+    }
+
+    public void setName(String n){
+        name = n;
+    }
+    public void setComment(String c){
+        comment = c;
+    }
+    public void setDate(){
```

Comment:
I'd maybe give this a more descriptive name since it doesn't take an argument, like `setDateToCurrentTime()` or add a comment so that it's obvious what's happening.